shell.client/src/main/java/org/metaborg/spoofax/shell/client/Repl.java

```diff
@@ -43,6 +44,9 @@
         this.editor = editor;
         this.display = display;
         this.invoker = invoker;
+
+        this.editor.setPrompt(new StyledText(Color.RED, "[In ]: "));
```

Comment:
Since we haven't discussed looks yet, i just picked a random color to see if stuff works.